x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -34,6 +34,17 @@ import { getLatestAvailableVersion } from './versions';
 
 const INACTIVE_AGENT_CONDITION = `status:inactive OR status:unenrolled`;
 const ACTIVE_AGENT_CONDITION = `NOT (${INACTIVE_AGENT_CONDITION})`;
+const AGENT_STATUS = [
```

Comment:
I wonder if we could do a refactor to avoid repeating the statuses from here: https://github.com/elastic/kibana/blob/80c655912e5723a1cc4389aa4dd8263d5b43c657/x-pack/plugins/fleet/common/types/models/agent.ts#L20

Something similar was done here:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/common/constants/agent.ts#L29-L37
https://github.com/elastic/kibana/blob/80c655912e5723a1cc4389aa4dd8263d5b43c657/x-pack/plugins/fleet/common/types/models/agent.ts#L62-L63